### PR TITLE
Adds provision_config method

### PR DESCRIPTION
**Pull request by joewesch**
This PR takes over from #63

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -25,7 +25,7 @@ python = "^3.7"
 nornir = "^3.0.0"
 requests = "^2.25.1"
 nornir-utils = "^0"
-nornir-napalm = "^0"
+nornir-napalm = "^0.4.0"
```
Comment (jvanderaa) [inline pyproject.toml:28]: With this, if there is a Nornir-Napalm 0.5.0 released, we won't get it. Is this intended? @joewesch

**Review by jvanderaa — APPROVED** [review]
LG pending other comment.

**Review by jvanderaa — APPROVED** [review]
LG pending other comment.

File: pyproject.toml (hunk 1 above)
Comment (joewesch) [reply pyproject.toml:28]: I just need the minimum to be 0.4.0. I always forget about that caveat. I'll update it.

**Pull request merged**